src/packageurl/contrib/django/utils.py

```diff
@@ -33,6 +35,11 @@ def purl_to_lookups(purl, encode=True):
     Return a lookups dict built from the provided `purl` string.
     Those lookups can be used as QuerySet filters.
     """
+    warnings.warn(
+        "`purl_to_lookups` is moved from packageurl.contrib.django.models"
+        " to packageurl.contrib.django.utils",
+        DeprecationWarning,
+    )
```

Comment:
We should not warn here.

Warning should only be raised when the following import are made:
- `from packageurl.contrib.django.models import purl_to_lookups`
- `from packageurl.contrib.django.models import without_empty_values`

Why warning one that is properly importing from utils?

Reply:
Yeah I messed up that commit, made some changes in a new commit